src/platform/plugins/shared/dashboard/public/dashboard_api/performance/query_performance_tracking.ts

```diff
@@ -123,8 +131,52 @@ function reportPerformanceMetrics({
 }) {
   const duration =
     loadType === 'dashboardSubsequentLoad' ? timeToData : Math.max(timeToData, totalLoadTime);
+  const groupedPerformanceMarkers = getPerformanceTrackersGroupedById(
```

Comment:
How about renaming "groupedPerformanceMarkers" => "panelPerformanceMarkers". Since these are grouped by panel, I think having the word "panel" in the name is more descriptive then the word "grouped", as "grouped" does not really tell you what these are.